tdrs-backend/tdpservice/core/custom_session_engine.py

```diff
@@ -0,0 +1,59 @@
+from django.contrib.sessions.backends import signed_cookies
+from django.core import signing
+import datetime
+
+class SessionStore(signed_cookies.SessionStore):
+    """Custom session engine for TDP."""
+
+    def __init__(self, session_key=None):
+        """Initialize the custom session engine."""
+        super().__init__(session_key)
+
+    def load(self):
+        """Load the session data from the database."""
+        # get self.session
+        #session_data = super().load()
+        """
+        Load the data from the key itself instead of fetching from some
+        external data store. Opposite of _get_session_key(), raise BadSignature
+        if signature fails.
+        """
+
+
+        try:
+            return signing.loads(
+                self.session_key,
+                serializer=self.serializer,
+                # This doesn't handle non-default expiry dates, see #19201
+                max_age=datetime.timedelta(days=1), #self.get_session_cookie_age(), #
```

Comment:
we are only overriding max_age. By default `signed_cookies` assigns default `SESSION_COOKIE_AGE`, which will expire the session. We cannot use the default session `set_expiry` for signed sessions, unless we unsign and re-sign the session with every request which is more expensive.